internal/current_target.go

```diff
@@ -15,41 +13,16 @@ type CurrentTarget struct {
 }
 
 func LoadCurrentTarget(currentTargeFile string) (CurrentTarget, error) {
-	cur := CurrentTarget{}
-
-	file, err := os.Open(currentTargeFile)
+	var cur CurrentTarget
+	ctToml, err := toml.LoadFile(currentTargeFile)
 	if err != nil {
-		return cur, fmt.Errorf("Unable to read current-target file: %w", err)
+		return cur, err
 	}
-
-	scanner := bufio.NewScanner(file)
-	for scanner.Scan() {
-		line := scanner.Text()
-		line = strings.TrimSpace(line)
-		if len(line) != 0 && line[0] != '#' {
-			parts := strings.SplitN(line, "=", 2)
-			if len(parts) != 2 {
-				return cur, fmt.Errorf("Invalid line(missing =): %s", line)
-			}
-			if parts[0] == "TARGET_NAME" || parts[0] == "CUSTOM_VERSION" {
-				// Don't bother validating the value unless its one we care about
-				if len(parts[1]) < 3 {
-					return cur, fmt.Errorf("Invalid line(bad value): %s", line)
-				}
-				if parts[1][0] != '"' || parts[1][len(parts[1])-1] != '"' {
-					return cur, fmt.Errorf("Invalid line(bad value): %s", line)
-				}
-				value := parts[1][1 : len(parts[1])-1]
-				if parts[0] == "TARGET_NAME" {
-					cur.Name = value
-				} else {
-					cur.Version, err = strconv.Atoi(value)
-					if err != nil {
-						return cur, fmt.Errorf("Invalid CURRENT_VERSION(%s): %w", value, err)
-					}
-				}
-			}
-		}
+	cur.Name = ctToml.Get("TARGET_NAME").(string)
+	verStr := ctToml.Get("CUSTOM_VERSION").(string)
```

Comment:
These will panic if the toml library return non-string value.
I'd recommend smth like below:
```
if cur.Name, ok = ctToml.Get("TARGET_NAME").(string); !ok {
    return errors.New("Cannot parse TARGET_NAME")
}
```